README.md

```diff
@@ -7,30 +7,33 @@ See an overview of the flake outputs by running
 
 ## 🗃️ What's in here?
 
-Package                                                     | Description
-------------------------------------------------------------|------------
-[`faf-client`](./pkgs/faf-client)                           | Forged Alliance Forever client (multiple packages)
-[`osu-lazer-bin`](./pkgs/osu-lazer-bin)                     | osu! lazer, extracted from the official AppImage
-[`osu-stable`](./pkgs/osu-stable)                           | osu! stable version
-`rocket-league`                                             | Rocket League from Epic Games
-[`star-citizen`](./pkgs/star-citizen)                       | Star Citizen
-[`technic-launcher`](./pkgs/technic-launcher)               | Technic Launcher
-[`wine-discord-ipc-bridge`](./pkgs/wine-discord-ipc-bridge) | Wine-Discord RPC Bridge
-[`wine`](./pkgs/wine)                                       | Multiple Wine packages
-[`winestreamproxy`](./pkgs/winestreamproxy)                 | Wine-Discord RPC (broken)
-[`proton-ge`](./pkgs/proton-ge)                             | Custom build of Proton with the most recent bleeding-edge Proton Experimental WINE
-
-* `legendaryBuilder` is a function that installs games with `legendary-gl`. You
-are expected to log in before using it, with `legendary auth`.
-The function takes an attrset containing at least the attrset `games` which
-includes the games you want installed. Optionally, you can set an `opts`
-attrset that will set the options you set inside for all games listed.
-You can find a usage example in [example.nix](./example.nix).
+| Package                                                     | Description                                                                                            |
```

Comment:
Can you remove the prefixing/suffixing `|`? They'll be annoying to change everytime a longer description is added.

Reply:
Could I ask you to do it? My editor does it instantly thanks to prettier